gprofiler/metadata/application_identifiers.py

```diff
@@ -15,7 +15,11 @@
 from gprofiler.exceptions import CalledProcessError
 from gprofiler.log import get_logger_adapter
 from gprofiler.metadata.enrichment import EnrichmentOptions
-from gprofiler.profilers.java import jattach_path
+from gprofiler.platform import is_windows
```

Comment:
Suggestion - move relevant Java code to `gprofiler/metadata/application_identifier_java.py`, and import it & add to `_IDENTIFIERS_MAP` iff Windows?